operations/mimir-mixin/dashboards/dashboard-utils.libsonnet

```diff
@@ -513,43 +513,59 @@ local utils = import 'mixin-utils/utils.libsonnet';
+          |||
+            keda_metrics_adapter_scaler_metrics_value{metric=~".*cpu.*"}
+            /
+            on(metric, cluster) group_left label_replace(
+                kube_horizontalpodautoscaler_spec_target_metric{%(namespace)s, horizontalpodautoscaler=~"%(hpa_name)s"},
+                "metric", "$1", "metric_name", "(.+)"
+            )
+          ||| % {
+            hpa_name: $._config.autoscaling[component].hpa_name,
+            namespace: $.namespaceMatcher(),
+          },
         ], [
-          'Scaling metric',
-          'Target per replica',
+          '{{ scaledObject }}',
         ]
       ) +
       $.panelDescription(
         title,
         |||
-          This panel shows the result of the query used as scaling metric and target/threshold used.
-          The desired number of replicas is computed by HPA as: <scaling metric> / <target per replica>.
+          This panel shows the scaling metric exposed by KEDA divided by the target/threshold used.
+          It should represent the desired number of replicas, ignoring the min/max constraints applied later.
         |||
       ) +
       $.panelAxisPlacement('Target per replica', 'right'),
     )
     .addPanel(
-      local title = 'Scaling metric (Memory)';
+      local title = 'Scaling metric (memory): Desired replicas';
       $.panel(title) +
       $.queryPanel(
         [
-          $.filterKedaMetricByHPA('keda_metrics_adapter_scaler_metrics_value{metric=~".*memory.*"}', $._config.autoscaling[component].hpa_name),
-          'kube_horizontalpodautoscaler_spec_target_metric{%s, horizontalpodautoscaler=~"%s", metric_name=~".*memory.*"}' % [$.namespaceMatcher(), $._config.autoscaling[component].hpa_name],
+          |||
+            keda_metrics_adapter_scaler_metrics_value{metric=~".*memory.*"}
+            /
+            on(metric, cluster) group_left label_replace(
+                kube_horizontalpodautoscaler_spec_target_metric{%(namespace)s, horizontalpodautoscaler=~"%(hpa_name)s"},
+                "metric", "$1", "metric_name", "(.+)"
+            )
+          ||| % {
+            hpa_name: $._config.autoscaling[component].hpa_name,
+            namespace: $.namespaceMatcher(),
+          },
         ], [
-          'Scaling metric',
-          'Target per replica',
+          '{{ scaledObject }}',
         ]
       ) +
       $.panelDescription(
         title,
         |||
-          This panel shows the result of the query used as scaling metric and target/threshold used.
-          The desired number of replicas is computed by HPA as: <scaling metric> / <target per replica>.
+          This panel shows the scaling metric exposed by KEDA divided by the target/threshold used.
+          It should represent the desired number of replicas, ignoring the min/max constraints applied later.
         |||
       ) +
       $.panelAxisPlacement('Target per replica', 'right') +
```

Comment:
Since we don't want the bytes unit anymore, please remove the following:

```
  {
        yaxes: std.mapWithIndex(
          // Set the "bytes" unit to the right axis.
          function(idx, axis) axis + (if idx == 1 then { format: 'bytes' } else {}),
          $.yaxes('bytes')
        ),
      },
```

Reply:
@pracucci aha, that's where it comes from!

Reply:
Thanks, done.